metadata-service/factories/src/main/resources/application.yml

```diff
@@ -82,6 +82,10 @@ graphService:
 searchService:
   resultBatchSize: ${SEARCH_SERVICE_BATCH_SIZE:100}
   enableCache: ${SEARCH_SERVICE_ENABLE_CACHE:false}
+  cacheImplementation: ${SEARCH_SERVICE_CACHE_IMPLEMENTATION:caffeine}
```

Comment:
minor minor: in terms of config layout, what if we did this
```
cache:
    type: "hazelcase"
    config:
         hazelcast:
                 serviceName: .... same
```

this may read a bit easier. thoughts?

Reply:
Agreed, I had it at the upper level to be in line with the enableCache variable, but I think it's fine to start a new pattern here.